src/main/java/com/aws/greengrass/clientdevices/auth/configuration/RuntimeConfiguration.java

```diff
@@ -62,8 +64,24 @@ public void updateCACertificates(List<String> caCerts) {
      */
     public void updateCAPassphrase(String passphrase) {
         Topic caPassphrase = config.lookup(CA_PASSPHRASE_KEY);
-        // TODO: This passphrase needs to be encrypted prior to storing in TLOG
         caPassphrase.withValue(passphrase);
     }
 
+    /**
+     * Get Certificate Registry configuration topics.
+     *
+     * @return Topics
+     */
+    public Topics getCertRegistry() {
```

Comment:
We could return directly the Topics but in the other `getter` methods we are returning a primitive data type or a list.  (just the coerced value). We should decide if we want to make all return the topics or primitive values to keep the api consistent.